### PR TITLE
New bridge flow

### PR DESCRIPTION
**Pull request by rjriel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,13 +132,14 @@ for the API Key and click `submit`. The API call will be executed and the data w
...
-First we initiate the widget using your `public_key`. 
+First we make a call to our API to request a bridge token from Citadel and use this bridge token to initiate the Citadel Bridge.
 
 ```
+        const bridgeToken = await apiRequests.getBridgeToken()
         const bridge = CitadelBridge.init({
           clientName: 'Citadel Quickstart',
-          companyMappingId: null,
-          key: '{{ public_key }}',
+          companyMappingId: bridgeToken.settings.company_mapping_id,
```
Comment (chetvil) [inline README.md:141]: mhm

File: node/README.md [hunk 2]
```diff
@@ -78,40 +76,101 @@ Click exit icon at the top right of the widget and you'll see the empty form.
...
+  })
+```
+## <a id="step-3"></a>3. :computer: runs `CitadelBridge.init` with `bridge_token`
 ```
   const bridge = CitadelBridge.init({
-    clientName: 'Citadel Quickstart',
-    companyMappingId: null,
-    key: '{{public_key}}',
+    clientName: 'Citadel NodeJS Quickstart',
+    companyMappingId: bridgeToken.settings.company_mapping_id,
```
Comment (chetvil) [inline node/README.md:155]: del

File: README.md (hunk 1 above)
Comment (rjriel) [reply README.md:141]: yup yup, just got the new admin/bridge flow stuff in, working on doc stuff today

File: node/README.md (hunk 2 above)
Comment (rjriel) [reply node/README.md:155]: same as above

**Review by chetvil — APPROVED** [review]

**Pull request merged**